KwonSeongMin/KwonSeongMin_week8/11721.py

```diff
@@ -0,0 +1,7 @@
+#https://www.acmicpc.net/problem/11721
+
+l = input()
+for i in range(len(l)):
```

Comment:
```suggestion
for i in range(0, len(l), 10):
```
이런식으로 step을 정해주면 해당 step 만큼 증가해서 코드를 더 간결하게 작성하실 수 있을거에요